progfun/funcional_teoria/index.html

```diff
@@ -572,7 +588,7 @@ <h3 class="text-center">Seguridad</h3>
                             el codigo que estemos creando, y
                             a que no tenemos que preocuparnos por cambios que puedan sufrir los objetos a lo largo del
                             programa. Como ventaja adicional,
-                            los objetos que son inmutables, se vuelven automaticamente segutos en el manejo de hilos (o
+                            los objetos que son inmutables, se vuelven automaticamente seguros en el manejo de hilos (o
```

Comment:
```suggestion
                            los objetos que son inmutables, se vuelven automáticamente seguros en el manejo de hilos (o
```
Revisar otros errores de ortografía presentes en este párrafo (Característica*, código*, ningún*).